src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/configs/CacheConfig.java

```diff
@@ -22,21 +22,41 @@
 
     private Double fieldDataCacheSizeThreshold;
     private Double shardRequestCacheSizeThreshold;
+    private Integer fieldDataCollectorTimePeriodInSec;
+    private Integer shardRequestCollectorTimePeriodInSec;
 
     public static final double DEFAULT_FIELD_DATA_CACHE_SIZE_THRESHOLD = 0.8;
     public static final double DEFAULT_SHARD_REQUEST_CACHE_SIZE_THRESHOLD = 0.9;
+    public static final int DEFAULT_FIELD_DATA_COLLECTOR_TIME_PERIOD_IN_SEC = 300;
+    public static final int DEFAULT_SHARD_REQUEST_COLLECTOR_TIME_PERIOD_IN_SEC = 300;
 
     public CacheConfig(final RcaConf rcaConf) {
```

Comment:
can we add a java doc here to explain the meaning of each config setting ?

Reply:
Added